metrics/exclude_accounts.txt

```diff
@@ -0,0 +1,180 @@
+# These are accounts to be excluded from metrics query results.
+# These are Red Hat accounts that may skew results because of
+# the amount these accounts are used.
+#
+# The list of accounts with more details can be found here:
+# https://docs.engineering.redhat.com/display/VMaaS/Red+Hat+accounts
```

Comment:
Any idea how to keep this file in sync with the page? :-)

Reply:
I have the file with the accounts in numerical order.  When the code runs, it spits out all the excluded accounts in alphabetical order.  If we reorganize the page to use one of these orders or somehow make it so we can export the account list from the doc so we can use command line tools to sort it, we can do comparison to see what needs to be updated.